cli/dcoscli/package/main.py

```diff
@@ -4,6 +4,10 @@
 
 import docopt
 import pkg_resources
+try:
+    import yaml
+except ImportError:
+    yaml = None
```

Comment:
We don't have any output in yaml right now, just JSON. So for consistency we should remove the yaml option for now.